test/make.sh

```diff
@@ -14,63 +14,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This function checks to make sure that every
-# shebang has a '- e' flag, which causes it
-# to exit on error
-function check_bash() {
-find . -name "*.sh" | while IFS= read -d '' -r file;
-do
-  if [[ "$file" != *"bash -e"* ]];
-  then
-    echo "$file is missing shebang with -e";
-    exit 1;
-  fi;
-done;
+# find_files is a helper to exclude .git directories and match only regular
```

Comment:
Should this also exclude `.terraform` directories?

Reply:
Yes, it should.  Thanks for thinking of it, will add.

Reply:
@aaron-lane Added the behavior of excluding `.terraform` directories.  Also modified the `make check_headers` task to use `find_files` because it was failing on files inside of .terraform directories for me.